### PR TITLE
[WIP] DOC changing the doc in plot_glcm

### LINKED ISSUE #2469: DOCS: GLCM example hard to follow
In the GLCM Texture Features example, I find the following sentence popping out by not being understandable:

> For each patch, a GLCM with a horizontal offset of 5 is computed.

What is unclear:
* What means 'horizontal'? 
* Offset from what?

I'm looking at this page:
http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_glcm.html#sphx-glr-auto-examples-features-detection-plot-glcm-py

### PR DESCRIPTION
**Pull request by mrastgoo**
## Description
[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
closes #2469 

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](http://scikit-image.org/docs/dev/contribute.html)]

## References
[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: doc/examples/features_detection/plot_glcm.py [hunk 1]
```diff
@@ -4,19 +4,24 @@
...
+co-occurrence matrices (GLCMs) [1]_. A GLCM is a histogram of co-occurring
 greyscale values at a given offset over an image.
 
 In this example, samples of two different textures are extracted from
 an image: grassy areas and sky areas. For each patch, a GLCM with
-a horizontal offset of 5 is computed. Next, two features of the
-GLCM matrices are computed: dissimilarity and correlation. These are
-plotted to illustrate that the classes form clusters in feature space.
-
+a horizontal offset of 5 (`distance=[5]` and `angles=[0]`) is computed.
```
Comment (sciunto) [inline doc/examples/features_detection/plot_glcm.py]: Around line 53, then I think it is useful to declare variables and put a comment.

File: doc/examples/features_detection/plot_glcm.py [hunk 2]
```diff
@@ -4,19 +4,24 @@
...
+clusters in feature space.
 In a typical classification problem, the final step (not included in
 this example) would be to train a classifier, such as logistic
 regression, to label image patches from new images.
 
+References
+----------
+.. [1] Haralick, RM.; Shanmugam, K., 
+       "Textural features for image classification"
+       IEEE Transactions on systems, man, and cybernetics 6 (1973): 610-621.
```
Comment (sciunto) [inline doc/examples/features_detection/plot_glcm.py]: This reference does not appear in the dosctring of greycomatrix. If this reference is relevant, this should be homogenized at this level as well.

Alos, It could be nice to add the DOI with the format `DOI:`

**Comment by emmanuelle** [comment]
Hi @mrastgoo I'd like to merge this PR so I pushed a commit to your branch, I hope this is OK with you! @sciunto ready for another review ;-)
